tests/visco_plastic_yield_plastic_strain_weakening.prm

```diff
@@ -1,133 +1,5 @@
-# Global parameters
-set Dimension                              = 2
-set Start time                             = 0
-set End time                               = 0
-set Use years in output instead of seconds = true
-set Nonlinear solver scheme                = single Advection, iterated Stokes
-set Max nonlinear iterations               = 1
-set Output directory                       = visco_plastic_yield_plastic_strain_weakening
-set Timing output frequency                = 1
-    
-subsection Solver parameters
-  subsection Stokes solver parameters
-    set Number of cheap Stokes solver steps = 0
-  end
-end
-
-# Model geometry (100x100 km, 10 km spacing)
-subsection Geometry model
-  set Model name = box
-  subsection Box
-    set X repetitions = 10
-    set Y repetitions = 10
-    set X extent      = 100e3
-    set Y extent      = 100e3
-  end
-end
-
-# Mesh refinement specifications 
-subsection Mesh refinement
-  set Initial adaptive refinement        = 0
-  set Initial global refinement          = 0
-  set Time steps between mesh refinement = 0
-end
-
-
-# Boundary classifications (fixed T boundaries, prescribed velocity) 
-# The parameters below this comment were created by the update script
-# as replacement for the old 'Model settings' subsection. They can be
-# safely merged with any existing subsections with the same name.
-
-subsection Boundary temperature model
-  set Fixed temperature boundary indicators   = bottom, top, left, right
-end
-
-subsection Boundary velocity model
-  set Prescribed velocity boundary indicators = bottom y: function, top y: function, left x: function, right x: function
-end
 
-# Velocity on boundaries characterized by functions
-subsection Boundary velocity model
-  subsection Function
-    set Variable names      = x,y
-    set Function constants  = m=0.0005, year=1
-    set Function expression = if (x<50e3 , -1*m/year, 1*m/year); if (y<50e3 , 1*m/year, -1*m/year);
-  end
-end
+include $ASPECT_SOURCE_DIR/tests/visco_plastic_yield_plastic_strain_weakening.prm
```

Comment:
Ok, I'm a bit confused here. The include here is referring to the file it is located in. I am just not reading the file names correctly?

Reply:
Sorry, thanks for noticing that! My fault, thought I had fixed that before pushing... I was not reading filenames correctly yesterday evening, all so similar